### ISSUE TITLE
Updated and lost ALL tabs

### ISSUE THREAD

**Issue by TheMightyDevil** [greatsuspender/thegreatsuspender#1151]
Got a nice TAB mentionning an update for the Great Suspender. Clicked for update. No warning, no nothing, it just closed Chrome, never restarted it and had to open it manually. LOST all my tabs. Thank you Great Suspender. What gives!!?

**Comment by jeffreytigch** [comment]
Did you try Ctrl + Shift + T (or CMD + Shift + T on Mac)? Sounds like it crashed, mine updated just fine.

**Comment by TheMightyDevil** [comment]
Had a crazy amount of tabs...I know, I know, my bad...turns out, the Suspender did it's job! Swallowing back all the bad things I said in the last minutes.

Went to the Suspender's icon in Chrome and under, Session Management, found history of Recent Sessions and reloaded my session in this morning's date.

ALL tabs are back! 

Really just whished for a pop up or warning when I clicked UPDATE for the Suspender. Advisory mentionning what will happen next and where to find recent sessions afterwards.

Solved for me.
Thank you!
